### ISSUE TITLE
RHEL8 changes

### ISSUE THREAD

**Issue by ctramnitz** [poise/poise-python#139]
RHEL8 (beta) doesn't have a versioned "python" package anymore but only "python2" and "python36" packages. Also, they come from the appstream repo now, not from scl or system anymore.

That seems to break the current provider detection logic and may require some adjustments.

**Comment by coderanger** [comment]
Yeah, this will probably require writing a new provider specifically for RHEL8. As this project is in maintenance mode, I would encourage you to write that in your own wrapper cookbook for now, I wouldn't even attempt it until after the 8.0 release and even then it seems unlikely given my priorities have shifted away from Chef.
